### PR TITLE
[JENKINS-60167] - Fix AtomicFileWriter performance issue on CephFS in case of Empty File creation

### PR DESCRIPTION
**Pull request by bulanovk**
Fix for https://issues.jenkins-ci.org/browse/JENKINS-60167

Need to add StandardOpenOption.CREATE when creating FileChannelWriter to avoid full fs cache flush and 5 second operation Create Empty File(ex RunIdMigration scenario) on CephFS

### Suggested changelog entry

* Fix AtomicFileWriter performance issue on CephFS in case of Empty File creation

### CODE REVIEW

File: core/src/main/java/hudson/util/AtomicFileWriter.java
Comment (alecharp) [inline core/src/main/java/hudson/util/AtomicFileWriter.java]:
```suggestion
        core = new FileChannelWriter(tmpPath, charset, integrityOnFlush, integrityOnClose, StandardOpenOption.WRITE, StandardOpenOption.CREATE);
```

**Review by alecharp — APPROVED** [review]
I got worry when I saw the constructor was deprecated but it seems OK. Waiting for the CI to validate it.

**Review by varyvol — APPROVED** [review]

File: core/src/main/java/hudson/util/AtomicFileWriter.java
Comment (bulanovk) [reply core/src/main/java/hudson/util/AtomicFileWriter.java]: Fix white space, and squash commits

**Review by oleg-nenashev — APPROVED** [review]
The change looks right. I wonder what is the potential impact on other platforms, but the flag indeed makes sense for this logic.

Will merge it next  week if no negative feedback

**Pull request merged**
